src/components/Box/Box.tsx

```diff
@@ -0,0 +1,18 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { HTMLAttributes } from 'react';
+
+export type BoxProps = {
+    'data-test-id'?: string;
+} & HTMLAttributes<HTMLDivElement>;
+
+export const BOX_TEST_ID = 'fondue-box';
+
+export const Box = ({ children, 'data-test-id': dataTestId = BOX_TEST_ID, ...props }: BoxProps) => {
```

Comment:
We must provide a default value for `data-test-id` and explicitly define it since `data-*` props are defined by the user, of course other `data` props can be passed based on HTMLAttributes...